src/AppBundle/Resources/translations/user-account.en.yml

```diff
@@ -1,8 +1,9 @@
 h1Title:
     client-edit: "Account: %client%'s details"
     deputy-edit: "Account: Your details"
-    change-password: "Account: Change password"    
+    change-password: "Account: Change password"
+    add-account: "Add a user account"
```

Comment:
@Biggsen to decide if a different template is needed here. I'd suggest so, the user-account yml is used in many other views and might confuse Content editor